src/Database/Esqueleto/Record.hs

```diff
@@ -935,9 +935,9 @@ sqlMaybeSelectProcessRowDec RecordInfo {..} = do
       ]
   where
     wrapJust x = case x of
-      AppT (ConT ((==) ''Entity -> True)) _innerType -> id
       ((ConT ((==) ''Maybe -> True)) `AppT` _inner) -> AppE (ConE 'Just)
       (ConT _) -> id
+      (AppT _ _) -> id
       _ -> error $ show x
```

Comment:
Do we want to keep the `error` here? I feel like we may want to just have `id` as the fallback case instead of erroring on unknown constructors